core/src/main/java/org/apache/struts2/interceptor/ParameterAware.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.struts2.interceptor;
+
+import org.apache.struts2.dispatcher.HttpParameters;
+
+import java.util.Map;
+
+import static java.util.stream.Collectors.toMap;
+
+@Deprecated
+public interface ParameterAware extends org.apache.struts2.action.ParametersAware {
+
+    void setParameters(Map map);
```

Comment:
To match the existing signature, which caused build issues for us, this needs to be:

setParameters(Map<String, String[]> parameters);

Reply:
Ah good catch, not sure how I managed to drop the generic types here

Reply:
I did check the other files in the review and it was just this one.

We were able to build fine and it's going through out automation, what we have anyway, tonight.